docs/build.sh

```diff
@@ -93,8 +93,11 @@ function generate_api_rst() {
 
   # Fill in boiler plate for extensions that have google.protobuf.Empty as their
   # config.
-  bazel run "${BAZEL_BUILD_OPTIONS[@]}" //tools/protodoc:generate_empty \
-    "${PWD}"/docs/empty_extensions.json "${GENERATED_RST_DIR}/api-${API_VERSION}"/config
+  if [[ "${API_VERSION}" != "v2" ]]
```

Comment:
Can we just not call this function for v2? I guess we are doing it for some reason down below? Can you add a few comments?

Reply:
Yep, we want to be able to link from previous release notes for now, as it's a major cleanup effort to remove those anchors (or have them point at some earlier external link).